airflow/ui/src/api/index.ts

```diff
@@ -39,6 +39,23 @@ export function useDags() {
   );
 }
 
+export function useDagRuns(dagId: Dag['dagId'], dateMin?: string) {
+  return useQuery<DagRunsResponse, Error>(
+    ['dagRun', dagId],
+    (): Promise<DagRunsResponse> => axios.get(`dags/${dagId}/dagRuns${dateMin ? `?start_date_gte=${dateMin}` : ''}`),
```

Comment:
We probably want some functions to handle the formatting of url params but that's not needed now.